### PR TITLE
Add support for PyJwt_2_0_0

### PR DESCRIPTION
**Pull request by rahulraina7**
## Goal
PyJWT just released v2.0.0 which breaks sanic-jwt due to https://github.com/jpadilla/pyjwt/issues/529

## Approach
Type checking before returning access token.

### CODE REVIEW

**Comment by ahopkins** [comment]
Seems like a simple change. Would you be interested to pull together a PR?

**Review by ahopkins — APPROVED** [review]

**Comment by ahopkins** [comment]
Looks like I need to fix the Travis builds. Will look into that next week and get this merged. Thanks for the PR

**Review by ahopkins — CHANGES_REQUESTED** [review]
Can you also update the requirements?

**Comment by rahulraina7** [comment]
> Can you also update the requirements?

Updated setup.py

**Review by ahopkins — APPROVED** [review]

**Comment by ahopkins** [comment]
Can you take a look at the failing tests?

**Comment by rahulraina7** [comment]
Hi @ahopkins: I have fixed the tests. The suite is failing due to some issues with coverage, probably some configuration issues. I will try to have a look but if you know what this is about then that's better

**Comment by ahopkins** [comment]
Awesome. I am not too worried about the codecov. Thanks for this  I will get the checks sorted out and merge.

**Comment by rahulraina7** [comment]
I have commented out the coverage combine command for now, since there aren't multiple coverage files to combine.

**Comment by stikks** [comment]
Happy new year guys. Any idea when this PR might get merged into master and released.

**Comment by rahulraina7** [comment]
@ahopkins : I am freezing this project dependency on sanic - 20.9.1. Due to https://github.com/huge-success/sanic/issues/1989 for now. Either I can make those changes you suggested there or wait for sanic for the patch.

**Comment by ahopkins** [comment]
I am pushing out a patch for that shortly.

**Comment by ahopkins** [comment]
See #1993

File: tox.ini [hunk 1]
```diff
@@ -24,7 +24,7 @@ deps =
     pytest-flakes
     pytest-pep8
     pytest-travis-fold
-    sanic
+    sanic==20.9.1
```
Comment (ahopkins) [inline tox.ini:27]: I am not going to merge this with a 20.9 dependency.

Just add this line to the top of `conftest.py` after the imports.

```python
Sanic.test_mode = True
```

**Review by ahopkins — CHANGES_REQUESTED** [review]
Small change to overcome the name registry and then ready to merge.

File: tox.ini (hunk 1 above)
Comment (ahopkins) [reply tox.ini:27]: In the coming weeks, we will release: https://github.com/sanic-org/sanic-test

It will automatically set that.

**Review by ahopkins — APPROVED** [review]

**Comment by ahopkins** [comment]
Released. Thank you :clap: 

https://pypi.org/project/sanic-jwt/1.6.0/

**Pull request merged**